region/lowernorfair/west/Golden Torizo's Room.json

```diff
@@ -2077,8 +2115,19 @@
         "This can be done with Space Jump, a mid-air Spring Ball jump with HiJump, a spring wall, or a speedy jump with HiJump.",
         "All of these cases can be done with Golden Torizo either alive or dead."
       ]
+    },
+    {
+      "id": 8,
+      "name": "Safe Spot Kill (Lenient)",
+      "note": [
+        "The ability to get into the safe spot in the Golden Torizo fight, where Samus can attack while being safe from most of GT's attacks.",
+        "Samus is not able to shoot GT with Supers in this lenient safe spot and she must have fewer than 32 Missiles at all times.",
+        "To get into the safe spot near the door: Stand on the left side of the floor pillar for GT to jump back. Then stand to the right of the nearby background pillar.",
+        "This safe spot expects getting hit by GT before it jumps back. The space includes the less lenient spot, and also extends left 9 extra pixels."
+      ],
+      "devNote": "There is a 14 px window where Samus is safe from all of GT's attacks except the thrown Super attack and the attack that only occurs when Samus has too many Missiles."
```

Comment:
A comment about the bug spawner attack when he steps forward could fit here, since it can hit you still.